aten/src/ATen/native/Convolution.cpp

```diff
@@ -234,8 +234,8 @@ auto ConvParams::use_mkldnn(const at::Tensor& input) const -> bool {
     (input.options().backend() == at::Backend::CPU &&
      input.scalar_type() == kFloat && // only on CPU Float Tensors
      !is_dilated() && // doesn't support dilation
```

Comment:
should you also remove is_dilated check from here if it's actually supported by mkldnn?

Reply:
There has another PR to do it, see https://github.com/pytorch/pytorch/pull/40220.